06-checks-and-assets.md

```diff
@@ -33,6 +33,7 @@ If you want more details on a check, the `info` subcommand can help you out.
 === marketing-site
 Name:           check-http
 Interval:       10
+Cron:           0 30 \* \* \* \*
```

Comment:
We shouldn't allow setting both an interval and a cron schedule.

Reply:
It was my understanding of 1.x that you could set both schedules, but cron will take precedence in the presence of both. I can definitely change that here and in the logic if that's not the case!